src/automation_agent/review_provider.py

```
@@ -40,114 +47,249 @@ class LLMReviewProvider(ReviewProvider):
+                                return self._format_jules_review(review_text), metadata
+                            else:
+                                logger.warning("[JULES] Session completed but no review content found")
+                                error_dict = {
+                                    "success": False,
+                                    "error_type": "jules_empty_response",
+                                    "message": "Jules session completed but returned no review content"
+                                }
+                                return error_dict, {}
+                                
+                        except Exception as e:
+                            logger.error(f"[JULES] Failed to parse session response: {e}")
```

Ruff (0.14.7)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)